pages/facts.html

```diff
@@ -4,9 +4,71 @@
+      </li>
+      <li class="fact-li">
+        <h2 >0.1 Python is named after a TV show</h2>
+        <p >Python seems like an odd name for a programming language. There is no truth to the popular belief that Python is named after the famous non&ndash;venomous snake of the same name. It has a whole different story to tell. Guido was a huge fan of “Monty Python&sbquo;s Flying Circus”&comma; a comedy TV show on the BBC. Back in the 70s&comma; Monty Python was a renowned British comedy group that performed in British colonies. In honor of Monty Python&sbquo;s Flying Circus&comma; Guido decided to name the language Python&comma; since he was a big fan.</p>
+      </li>
+      <li class="fact-li">
+        <h2 >0.2 The Python programming language is older than Java</h2>
+        <p >Over the past decade&comma; Python has become a very popular programming language. It appears that Python has attracted and strengthened its listeners since it started out as a new language. <br> <br>
+        Here&sbquo;s the interesting truth: Python is a much older programming language than Java. The first Python version was released in 1991&comma; whereas the first Java version was released in 1995. Even though Python is 30 years old&comma; it still stands strong.</p>
+      </li>
+      <li class="fact-li">
+        <h2 >0.3 Python Anti&ndash;Gravity</h2>
+        <p >When you type import antigravity in the Python Interpreter&comma; Python gives you this easter egg. You will then be taken to a website with a comic&ndash;style image of a stick figure named Python flying. Check it out for yourself. Here&sbquo;s the import code you can run on your Python interpreter or console.</p>
+      </li>
+      <li class="fact-li">
+        <h2 >0.4 Special Slice Operator in Python</h2>
+        <p >The Python programming language has a special slice operator that makes it very unique compared to other programming languages. This operator can perform various tasks&comma; such as shortening a list&comma; getting the last and the first numbers&comma; etc. I have provided a code example below to better understand this interesting operator.</p>
+        <br>
+        <h5>Example Code:</h5>
+        <br>
+        <code id="example-code">
+          list = [5, 10, 15, 20, 25, 30,35] <br>
+          <br>
+          #Here we are shortening a list <br>
+          print(list[:-5]) # [5, 10] <br>
+          <br>
+          #Here we are reversing a list <br>
+          print(list[::-1]) <br> <br>
+
+          Output: 
+          <br>
+          
+          [5, 10] <br>
+          [35,30,25,20,15,10,5] <br>
+        </code> 
+      </li>
+    </ol>
+  </section>
+  <hr>
```

Comment:
The `hr` tag is self closing.
```suggestion
  <hr />
```

Reply:
The use of the [`hr` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/hr) here isn't semantic. 

The documentation says a couple of things:
"The `<hr>` element represents a thematic break between paragraph-level elements: for example, a change of scene in a story, or a shift of topic within a section."

"Historically, this has been presented as a horizontal rule or line. **While it may still be displayed as a horizontal rule in visual browsers, this element is now defined in semantic terms, rather than presentational terms, so if you wish to draw a horizontal line, you should do so using appropriate CSS.**"

If you're using an element and you're not sure if it's semantically appropriate for the structure of your page, it'll be important to review the documentation to check that you are using the element correctly.